### ISSUE TITLE
Can't open the .msk files directly

### ISSUE THREAD

**Issue by FastusBellrune** [olivierkes/manuskript#1251]
I don't know if it's because it remembers the previous version or what, but while I can open it from recent projects or by choosing the file from inside Manuskript, I can't open it from clicking on the .msk file directly. I tried changing the default program for the .msk extension, but it doesn't work.

![Screenshot 2023-12-14 193716](https://github.com/olivierkes/manuskript/assets/119754614/11368c61-f46c-46e2-8d91-bc88e0c29418)

**Comment by TheShadowOfHassen** [comment]
I've never been able to open a .msk file, and I've been using manuskript since 0.12 however, it's a good idea and should be added.

**Comment by FastusBellrune** [comment]
I could do it with version 0.15.0. I just had to make manuskript.exe the default program for opening the .msk extension.

**Comment by TheShadowOfHassen** [comment]
I'm on Pop!_OS/ Ubuntu, and I've never managed to. It could just be a Linux thing though
Also, you probably should update to 0.16.1 It came out today and is supposed to have some important fixes for windows (maybe it'd fix it?)

**Comment by FastusBellrune** [comment]
No, I just didn't notice. I'll download it now. And I don't know how it could work on Linux. It's been a while since I used it.

**Comment by FastusBellrune** [comment]
It's still not working in version 0.16.1.

**Comment by TheShadowOfHassen** [comment]
@TheJackiMonster?

**Comment by TheJackiMonster** [comment]
Manuskript supports opening .msk files when their path is provided as first argument (that is the usual way how to implement it on application level).

Opening files with a specific extension is not actually part of an application but the operating system. So if setting the default application for an extension does not work on Windows, I'd recommend asking Microsoft. Can't be fixed on application level.
